src/test/java/org/mastodon/mamut/feature/branch/BranchDepthFeatureComputerTest.java

```diff
@@ -0,0 +1,72 @@
+package org.mastodon.mamut.feature.branch;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.mastodon.feature.Feature;
+import org.mastodon.feature.FeatureProjection;
+import org.mastodon.feature.FeatureProjectionKey;
+import org.mastodon.feature.FeatureProjectionSpec;
+import org.mastodon.mamut.feature.MamutFeatureComputerService;
+import org.mastodon.mamut.feature.branch.exampleGraph.ExampleGraph1;
+import org.mastodon.mamut.feature.branch.exampleGraph.ExampleGraph2;
+import org.mastodon.mamut.feature.branch.exampleGraph.ExampleGraph3;
+import org.mastodon.mamut.model.branch.BranchSpot;
+import org.scijava.Context;
+
+public class BranchDepthFeatureComputerTest
+{
+	@Test
+	public void testComputeBranchDepth1()
+	{
+		try(Context context = new Context())
+		{
+			final MamutFeatureComputerService featureComputerService = context.getService( MamutFeatureComputerService.class );
+			ExampleGraph1 exampleGraph1 = new ExampleGraph1();
+			featureComputerService.setModel( exampleGraph1.getModel() );
+			Feature< BranchSpot > feature = (Feature< BranchSpot> ) featureComputerService.compute( BranchDepthFeature.SPEC ).get(BranchDepthFeature.SPEC);
+
+			final FeatureProjection< BranchSpot > projection = feature.project( FeatureProjectionKey.key( BranchDepthFeature.PROJECTION_SPEC ) );
+
+			assertEquals( 0, projection.value( exampleGraph1.getBranchSpot( exampleGraph1.spot0 ) ), 0 );
```

Comment:
Good ideas. I implemented all of them.
There is one drawback with the namingScheme branchSpotA when it comes to debugging:

![grafik](https://user-images.githubusercontent.com/10515534/210998547-40dda222-8889-4ca3-bd0c-c42c87131ee4.png)

There is a need for a mental mapping between branchSpot names (A, B, C) and BranchVertex numbers (0, 1, 2). But I think that can be neglected.

Reply:
Ok, that's an interesting way to think about this. Naming "bv(3)" "branchSpot3" appears very reasonable. At the same time these indices are implementation dependent. We might change the implementation of the ModelBranchGraph, and this can cause the indices to change. So "branchSpotA" is probably still better.